pom.xml

```diff
@@ -397,8 +397,9 @@
         <version.io.agroal>2.0</version.io.agroal>
         <version.io.micrometer>1.9.13</version.io.micrometer>
         <version.io.netty>4.1.100.Final</version.io.netty>
-        <version.io.opentelemetry.opentelemetry>1.20.0</version.io.opentelemetry.opentelemetry>
-        <version.io.opentelemetry.proto>0.19.0-alpha</version.io.opentelemetry.proto>
+        <version.io.opentelemetry.opentelemetry>1.29.0</version.io.opentelemetry.opentelemetry>
+        <version.io.opentelemetry.proto>0.20.0-alpha</version.io.opentelemetry.proto>
+        <version.io.opentelemetry.opentelemetry-semconv>1.21.0-alpha</version.io.opentelemetry.opentelemetry-semconv>
```

Comment:
Is there any indication these will come out of alpha? I know we've been shipping with alphas, but I'm just curious.

Reply:
I have not seen any indication, no.